webhook/aws/sns.go

```diff
@@ -81,7 +82,10 @@ func NewSNSServer(v *viper.Viper) (ss *SNSServer, err error) {
 
 // NewNotifier creates Notifier instance using the viper config
 func NewNotifier(v *viper.Viper) (Notifier, error) {
-	return NewSNSServer(v)
+	if v != nil {
+		return NewSNSServer(v)
+	}
+	return nil, nil	
```

Comment:
You can always return a non-nil `Notifier` instance, even if `v` is nil.

That allows clients to do `NewNotifier(nil)` to get a default instance.